components/FutureGraph.tsx

```diff
@@ -0,0 +1,318 @@
+import {
+  ResponsiveContainer,
+  YAxis,
+  AreaChart,
+  LabelList,
+  Area,
+  ReferenceLine,
+} from 'recharts';
+import useSWR, { Fetcher } from 'swr';
+import {
+  FutureThresholdLegendIcon,
+  UpArrowIcon,
+  DownArrowIcon,
+  LineCircle,
+} from './icons/future_threshold';
+interface ForecastDataPointProps {
```

Comment:
New line above this interface just to separate it from imports.

In terms of the names of these interfaces, since they aren't being used to type the props of a React component, and instead they're core data types our app is using from fetching and processing, I think we can remove the "Props" suffix from this and the ones below.

```suggestion

interface ForecastDataPoint {
```